agenthub/micro/agent.py

```diff
@@ -73,10 +73,13 @@ def step(self, state: State) -> Action:
             latest_user_message=last_user_message,
         )
         content = [TextContent(text=prompt)]
-        if last_image_urls:
+        if self.llm.vision_is_active() and last_image_urls:
             content.append(ImageContent(image_urls=last_image_urls))
         message = Message(role='user', content=content)
-        resp = self.llm.completion(messages=[message.model_dump()])
+        resp = self.llm.completion(
+            messages=self.llm.format_messages_for_llm(message),
+            temperature=0.0,
```

Comment:
Maybe we don't hardcode temperature here?

Reply:
hmm it's hardcoded elsewhere, too, iirc 🤔

Reply:
See also browsing_agent.py and codeact_swe_agent.py for 0.0